src/pluginscript_callbacks.lua

```diff
@@ -97,6 +98,14 @@ pluginscript_callbacks.script_init = wrap_callback(function(manifest, path, sour
 	manifest = ffi_cast('godot_pluginscript_script_manifest *', manifest)
 	path = tostring(ffi_cast('godot_string *', path))
 	source = tostring(ffi_cast('godot_string *', source))
+    local is_yue = path:sub(-4) == '.yue' 
```

Comment:
I have to fix tabbing on this as well (or most of the code rather).
I basically just check if the file that's being loaded ends with `.yue`, i'll use this property later

Reply:
Instead of adding specific extra code for Yuescript in `script_init`, maybe you could create a new callback `yuescript_init` that does the translation, then calls `script_init` for going through the actual import process. This would need a new C function hooked in the Yuescript PluginScript, but this shouldn't be a problem. Also the `source` and possibly `path` parameters could need to support both Lua strings or `godot_string`s. Maybe always receive Lua strings and create a `lua_script_init` for forwarding the `godot_string` already transformed, idk.